charts/asset-manager/templates/clamav-configmap.yaml

```
@@ -25,6 +25,7 @@ data:
     # (as opposed to UNIX socket).
     MaxFileSize 500M
     MaxScanSize 1200M
+    maxScanTime 240000
```

tiny nit: might as well capitalise it the same as the others 🤷 
```suggestion
    MaxScanTime 240000
```

---

dooooh Thank you Chris saved me pulling my hair out when i would have been troublshooting this :)